src/language-html/utils.js

```diff
@@ -662,7 +662,13 @@ function unescapeQuoteEntities(text) {
 
 // top-level elements (excluding <template>, <style> and <script>) in Vue SFC are considered custom block
 // See https://vue-loader.vuejs.org/spec.html for detail
-const vueRootElementsSet = new Set(["template", "style", "script", "html"]);
+const vueRootElementsSet = new Set([
+  "template",
+  "style",
+  "script",
+  "html",
+  "HTML",
```

Comment:
What about `HtMl`? :simle:

Reply:
I missed it. we should support it like current html formatting.

**Prettier 2.0.5**
[Playground link](https://prettier.io/playground/#N4Igxg9gdgLgprEAuEAeAEjAsgGQHwA6UABMagCYCWAbnqgPRW1EOa54gA0IEADjJWgBnZKACGAJwkQA7gAVJCESjEAbGWICeI7gCMJYsAGs4MAMq9DlKAHNkMCQFc43ABYwAtqoDqryvCFLMDgzJX8af01kcCEdEGshOAkYOQMbDzFkADM1RO4AKyEADwAhA2NTMzEPOBxrOGzclxBCorNrG1U4AEVHCHhG1TyQSwlEiWj3Ly4RiWsYb0pyGFdkAA4ABm5eaUTvA15onbhx6gbuAEc++FS+ZRAxIQBaKDg4cneZiTgrym-UsTpTJIHJDZqJDyUexOcEdLq9foNEFNbgwMS6RbLVZIABMqIMlFUHQAwhAPBloicAKwzRyJAAq6OUoOG1GcAEkoB9YGYwHN+ABBLlmGCaLqDRIAX0lQA)
```sh
--parser html
```

**Input:**
```html
<HtML>
  <div></div>
</HtML>
```

**Output:**
```html
<html>
  <div></div>
</html>

```